src/collection/components/CollectionOrBundleEdit.tsx

```diff
@@ -680,12 +680,32 @@ const CollectionOrBundleEdit: FunctionComponent<
 
 			if (collectionState.currentCollection) {
 				const newCollection = await CollectionService.updateCollection(
-					collectionState.initialCollection,
-					collectionState.currentCollection,
+					omit(collectionState.initialCollection, ['loms', 'contributors']),
+					omit(collectionState.currentCollection, ['loms', 'contributors']),
```

Comment:
it might be andy to have a single helper for this
stripCollectionFieldsBeforeInsertOrUpdate(collection)